src/style.css

```diff
@@ -1,3 +1,89 @@
+:root {
+  --font-text: "Roboto", sans-serif;
+  --font-size-small: 12px;
+  --bg-color-white: #fff;
+  --text-color-black: #000;
+  --text-color-blue: #00acdc;
```

Comment:
Better not to use color name in variable name
Example of better names: '--main-color, --hover-color'....